source/library/Patrol/Type/Event.hs

```diff
@@ -136,6 +135,7 @@ initial =
     { environment = Text.pack "production",
       level = Just Level.Error,
       platform = Just Platform.Haskell,
+      sdk = Just ClientSdkInfo.patrol,
```

Comment:
https://github.com/tfausak/patrol/pull/32#discussion_r2143058224

Not strictly necessary since the client SDK info comes from the authorization header, but probably a good idea to set this regardless.